environments/docker-compose.local.yml

```diff
@@ -0,0 +1,22 @@
+---
+version: "3.8"
+services:
+  nautobot:
+    command: "nautobot-server runserver 0.0.0.0:8080"
+    ports:
+      - "8080:8080"
```

Comment:
Why's this in/needed in `docker-compose.local.yml` rather than `docker-compose.base.yml`?

Reply:
To make it easier to create another compose file for a different environment, say dev or staging.